src/components/reportPage/StepUpload.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useEffect, useState } from "react";
+import "./batch-job.css";
+import { Container, Row, Col } from "react-bootstrap";
+import UploadBox from "./UploadBox";
+import "../mainPage/mainpage.css";
+import "./upload.css";
+import Alert from "react-bootstrap/Alert";
+
+const StepUpload = (props) => {
+  // File Upload Alert Messages
+  const [showAlert, setShowAlert] = useState(false);
+  const [alertMessage, setAlertMessage] = useState();
+  const FILE_LIMIT_WARNING =
+    "You have exceeded the maximum limit of " + props.fileLimit + " files.";
```

Comment:
[Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) are a bit nicer to use